apps/gp2-frontend/src/events/calendar/Calendars.tsx

```diff
@@ -1,26 +1,34 @@
-import { CalendarList, EventsCalendar } from '@asap-hub/react-components';
+import {
+  CalendarList,
+  EventsCalendar,
+  pixels,
+} from '@asap-hub/react-components';
 
 import { useCalendars } from './state';
 
+const { rem } = pixels;
+
 const Calendars: React.FC<Record<string, never>> = () => {
   const { items } = useCalendars();
   return (
-    <EventsCalendar calendars={items}>
-      <CalendarList
-        title="Subscribe to Working Groups on Calendar"
-        description="Below you can find the list of all working groups. Hitting subscribe will allow you to add them to your own personal calendar."
-        calendars={items.filter(
-          ({ projects, workingGroups }) =>
-            projects.length === 0 || workingGroups.length > 0,
-        )}
-        hideSupportText
-      />
-      <CalendarList
-        calendars={items.filter(({ projects }) => projects.length > 0)}
-        title="Subscribe to Projects on Calendar"
-        description="Below you can find the list of all projects. Hitting subscribe will allow you to add them to your own personal calendar."
-      />
-    </EventsCalendar>
+    <div style={{ margin: `${rem(48)} 0` }}>
```

Comment:
no stylings here, move this to the Page components that is using it, probably <EventsPage/>